app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly implement the **_confirmable module_** of the `devise` gem in the application. It is part of the project's requirement. You can include `:confirmable` in the modules allowed for the `devise` gem in this file. 👍

Reply:
Thank you.